Aztec/Classes/Private/Libxml2/Data/ElementNode.swift

```diff
@@ -903,6 +949,21 @@ extension Libxml2 {
             }
         }
 
+        func replace(range targetRange: NSRange, withNodeNamed nodeName: String, withAttributes attributes:[Attribute]) {
+            guard let textNode = lowestTextNodeWrapping(targetRange) else {
```

Comment:
Extra info for the issue:

- The new method should also return the ranges so we can avoid using `absoluteLocation()`.  I'd base it on `lowestBlockLevelElements(...)`.
- The problem with `absoluteLocation()` is that it works fine only if you call this method in the rootNode.